client/packages/system/src/IndicatorsDemographics/api/api.ts

```diff
@@ -36,38 +36,6 @@ const Parsers = {
     };
     return fields[sortBy.key] ?? DemographicProjectionSortFieldInput.Id;
   },
-  toInsertIndicator: (
```

Comment:
No longer need these parsers because we generate the types as the insertInput and updateInput rather than as a fragment on create